events/smkmeetup/index.html

```diff
@@ -32,6 +32,37 @@ <h2>Get ready for the 5th semi-annual <a href="https://twitter.com/search?f=twee
 
   <a href="https://ti.to/ckeys/seattle-mechanical-keyboard-meetup-summer-2018" class="button special fit">RSVP</a>
 
+  <a name="speakers"></a>
+  <hr />
+
+  <h3>Speakers</h3>
+  <p>Keyboard and keyboard related talks will begin in the morning. More speakers will revealed over the coming weeks.</p>
+
+    <section class="spotlight">
+      <div class="image"><a href="https://cassidoo.co"><img src="/images/speaker-cassidy-williams.jpg" alt="Cassidy Williams with keyboard" /></div><div class="content">
+        <h4>Cassidy Williams</a></h4>
+        <p>Cassidy is the Head of Developer Voice Programs at Amazon. She comes from startup land, previously working at L4 Digital, Clarifai, and Venmo, and graduated with a computer science degree from Iowa State University in 2014. She's had the honor of working with various organizations, including Hacker Fund as their Director of Outreach. She's active in the developer community and one of Glamour Magazine's 35 Women Under 35 Changing the Tech Industry and LinkedIn's Top Professionals 35 &amp; Under. As an avid speaker, Cassidy has participated in meetups and events including the Grace Hopper Celebration for Women in Computing, SXSW, the National Center for Women &amp; IT Summit, TEDx, the United Nations, and the U.S.A. Science and Engineering Festival. She loves mechanical keyboards, with her daily drivers in rotation between a Clueboard with Kailh Browns and DSA Royal Navy, a Varmilo VA68Mg with MX Browns and SA Abyss, a WhiteFox with 65g tactile Zealios, and a Welfare96 with BOX Browns and GMK Oblivion.</p>
+      </div>
+    </section>
+    <section class="spotlight">
+      <div class="image"><a href="https://www.tinymakesthings.com"><img src="/images/speaker-tiny.jpg" alt="Tiny with keyboard" /></div><div class="content">
+        <h4>Tiny</a></h4>
+        <p>Tinymakesthings started making keycaps in 2016. She specializes in sculpting custom clay caps but has recently started to venture into the resin casting space. She loves to share her passion for mechanical keyboards and often livestreams her creative process on Twitch.</p>
+      </div>
+    </section>
+    <section class="spotlight">
+      <div class="image"><a href="https://talljoe.com/about/"><img src="/images/speaker-joe-wasson.jpg" alt="Joe Wasson" /></div><div class="content">
+          <h4>Joe Wasson</a></h4>
+        <p>I am a software engineer and team lead. I started my career at Microsoft in the mid 90’s and have worked for a variety of companies large and small, most recently as the Director of Product for payments at Xero, Inc. As part of finding “a better way”, I have built and programmed a more than 2-dozen mechanical keyboards in various layouts and have contributed to the QMK Firmware project. I’m currently learning stenography in attempt to further decrease friction in the human/machine interface.</p>
```

Comment:
This feels weird, jumping from third to first person

Reply:
I agree. That’s how the speakers sent their bios. You think we should re-write the first person ones?

Reply:
I think we should tweak either the third person or first person ones so they match. Not particularly concerned with which ones.